### ISSUE TITLE
Revisit and revise training guide

### ISSUE THREAD

**Issue by o-ram** [cal-itp/customer-success#502]
Split out from issue #440. Will be paused while Caitlin reviews documentation to see what updates would be most effective.

**Comment by caitlinharper** [comment]
How do you want to handle feedback & edits? I met with Jess, Laney, and Olivia on July 3rd to go over my feedback, but not sure if you want to review and reject/accept edits yourselves or you just want me to do it — another option is that I could make all changes in “suggesting” mode and whoever wants to can accept/reject.

**Comment by laneymangan** [comment]
I like the idea of having the changes be made in suggesting mode, then we can review and accept/reject as a team once Olivia is back in office. Open to other opinions of course!

**Comment by caitlinharper** [comment]
Hey! Thanks for your speedy reply @laneymangan :D Was just moving my Slack messages in here so I didn't lose track of what I'm working on. 

I'm thinking Olivia will feel similarly (suggesting mode), but it's totally fine to wait until she comes back so she can weigh in before I do anything.

**Comment by o-ram** [comment]
hey! feedback/edits in suggesting mode is perfect. Makes reviewing very easy on our end :)

**Comment by laneymangan** [comment]
@caitlinharper welcome back! just circling back here--are we all good to approve/review your suggestions in the training guide or is there more to add?

**Comment by caitlinharper** [comment]
Thank you @laneymangan. I'm done! Please approve/review and if you have any questions at all, just comment and @ me!

**Comment by laneymangan** [comment]
Olivia and I touched base on this 11/12. Final reviews/edits will be done following CTA but before December holidays

**Comment by o-ram** [comment]
Laney and Olivia to revisit in 2025

**Comment by caitlinharper** [comment]
@laneymangan & @o-ram – this is all wrapped up in the AM handbook now, right? If I'm remembering correctly, then maybe we should close this, as in we did revisit and revise, and then the next stage of revision can be built out under the future AM Handbook work?

**Comment by o-ram** [comment]
@caitlinharper yes, that's right - the training guide did become the AM handbook. And while it still needs updating, I agree that this can be closed and those updates now fall under the AM handbook bucket. 

**For reference:** Closing the issue! We did revisit and revise and the training guide is now within the AM handbook (#1028). The next round of revisions will fall under the next round of handbook content updates currently being scoped out.
